ember-headlessui/addon/components/disclosure/button.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@glimmer/component';
+
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { guidFor } from '@ember/object/internals';
+
+
+export default class Button extends Component {
+  guid = `${guidFor(this)}-tailwindui-disclosure-button`;
```

Comment:
can we use the guid helper instead of having a backing class?

template-only components are more performant